cli/p2p_replicator_set.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/spf13/cobra"
+
+	"github.com/sourcenetwork/defradb/client"
+)
+
+func MakeP2PReplicatorSetCommand() *cobra.Command {
+	var collections []string
+	var cmd = &cobra.Command{
+		Use:   "set [-c, --collection] <peer>",
```

Comment:
todo: `[-c, --collection]` appears to be inconsistent with the rest of the CLI, I think it should be consistent.

Reply:
Are you referring to the naming or that it's a slice type?

Reply:
I forgot that this was a slice type, but I was referring to the existence of the shorthand.  However the way you have moved the document stuff into collection has removed the inconsistency (as there is no longer a `--collection` flag there)